packages/core/src/provider.ts

```diff
@@ -143,6 +143,7 @@ export const networkNames: { [chainId in number]?: string } = Object.freeze({
   43114: 'avalanche',
   42220: 'celo',
   44787: 'celo-alfajores',
+  11155420: 'op-sepolia',
   11155111: 'sepolia',
```

Comment:
IMO they should be sorted by chainId
```suggestion
  11155111: 'sepolia',
  11155420: 'op-sepolia',
```